app/models/bookmark.rb

```diff
@@ -32,6 +32,8 @@ def check_new_external_work
     end
   end
 
+  after_update :admin_hidden_bookmark_notification, if: :hidden_by_admin_changed?
```

Comment:
Can you move this to be one (or two) line(s) above the `before_destroy` callback below, please